src/inmanta/protocol/methods_v2.py

```diff
@@ -1599,6 +1602,14 @@ def discovered_resources_get_batch(
     :param sort: Return the results sorted according to the parameter value.
             The following sorting attributes are supported: 'discovered_resource_id'.
             The following orders are supported: 'asc', 'desc'
+    :param filter: Filter the list of returned resources.
+        Default behavior: return all discovered resources.
+        Filtering by 'resource_id' is supported:
+            - filter.resource_id=true: only return discovered resources that the orchestrator is already aware of i.e.
+            managed resources that make up the latest configuration model and orphaned resources (unpurged resources from
+            previous versions of the configuration model).
```

Comment:
I think this explanation is more complicated than required. I wouldn't use the term orphaned resource here, because it's ill-defined in this context.
```suggestion
            - filter.resource_id=true: only return discovered resources that the orchestrator is already aware of i.e.
            resources that are present in any configurationmodel of environment tid.
```